WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1173,7 +1187,7 @@ MailMsgBodyText             = color:@darken(SecondaryBrandingColor,44)@;
 MailMsgBodyText-fixed       = @FontFamily-fixed@
 MailMsgBodyText-html        = @FontFamily-default@
 
-MailConvExpanded            = @AppContainerBg@
```

Comment:
I have added this change in last PR: https://github.com/Zimbra/zm-web-client/pull/32/files#diff-16ae02866e4f63813de9a7825a1057e3, which is merged. And this line reverts that changes.

I think you may have to rebase branch zcs-823 with universal-ui first?

Let me know.

Thanks